FlowCrypt/Models/OrganisationalRule.swift

```diff
@@ -85,8 +85,7 @@ class OrganisationalRules {
         }
 
         if keyManagerUrlForPrivateKeys == nil {
-            assertionFailure("Wrong org rules config: using PRV_AUTOIMPORT_OR_AUTOGEN without key_manager_url")
-            return false
+            fatalError("Wrong org rules config: using PRV_AUTOIMPORT_OR_AUTOGEN without key_manager_url")
         }
```

Comment:
This is a bit complicated - here we work with OrgRules which are external input (from a remote server). They could theoretically be any combination, and it would not be a fault of the app or bug in the app. We could say that we only use `mustAutoImportOrAutogenPrvWithKeyManager` once we checked that `keyManagerUrlForPrivateKeys` exists, which is fair. 

I guess we can keep it how you did it. It is true that if we ever check `mustAutoImportOrAutogenPrvWithKeyManager` without checking `keyManagerUrlForPrivateKeys` or `isUsingKeyManager` first, it suggests wrong coding somewhere.